client/src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Link, NavLink } from 'react-router-dom';
+
+/**
+ * @class Navbar
+ * @classdesc returns a navbar component 
+ */
+export default class Navbar extends React.Component {
```

Comment:
This component will be better off a functional component.

Reply:
Oh yeah true. Thanks for the feedback.